rust_snuba/src/processors/replays.rs

```diff
@@ -188,6 +188,15 @@ pub fn deserialize_message(
                 ..Default::default()
             }]
         }
+        ReplayPayload::ViewedEvent(event) => {
+            vec![ReplayRow {
+                replay_id: replay_message.replay_id,
+                viewed_by_id: event.user_id,
+                event_hash: Uuid::from_u64_pair(0, event.user_id), // for dedup
+                timestamp: event.timestamp as u32,
+                ..Default::default()
```

Comment:
We definitely do not want to default the `project_id` field.  You can pull it from the outer scope.  Its defined on the `ReplayMessage` type.